src/utils/logger.py

```diff
@@ -0,0 +1,18 @@
+import logging
+
+
+def get_logger() -> logging.Logger:
+    """
+    Get the logger for the application and set it up
+    :return: logger
+    """
+    logger = logging.getLogger("lynx")
+    logger.setLevel(logging.DEBUG)
+    formatter = logging.Formatter('%(asctime)s - %(process)s - %(name)s - %(levelname)s - %(funcName)s - %(message)s')

```

Comment:
We don't need `%(name)s`, because we only have one logger name ("lynx"); `%(process)s`, because we only log in the main process and `%(funcName)s`, because the logs are unique and the function name can be easily found.
I propose format:
`logging.basicConfig(format="%(asctime)s [%(levelname)s]: %(message)s")`
It should produce sth like:
`2023-07-010 19:50:08,123 [INFO]: <message that is being logged>`

Reply:
I would only remove %(name)s and %(funcname)s, we have mutliprocess application, the runtime is executed by another process.

Reply:
Fair enough. Let's go with `logging.basicConfig(format="%(asctime)s [%(process)s] [%(levelname)s]: %(message)s")` then.

Reply:
The formatter `formatter = logging.Formatter('%(asctime)s [%(process)s] [%(levelname)s] - %(message)s')` has been added, so the thread can be